modules/va_facilities/app/controllers/va_facilities/v1/nearby_controller.rb

```diff
@@ -51,6 +52,10 @@ def set_default_format
         request.format = :json if params[:format].nil? && request.headers['HTTP_ACCEPT'].nil?
       end
 
+      def set_facility_type
```

Comment:
Do we need to run this as a before action if this endpoint is only ever going to return health facilities? Would it be more clear to pass in `'health'` to Bing/future Postgres query and not mess with params at all?

Reply:
I'm not a big fan of setting the param this way, but we need to validate the `services[]` param and need to know the facility type to do that. Setting the `type` param allows us to continue to utilize `validate_type_and_services_known` for this validation.

Reply:
Makes sense 👍